### DISCUSSION TITLE
More details for hotels, apartments and chalets

### DISCUSSION THREAD

**Discussion by mcliquid** [Helium314/SCEE#646] [Ideas]
I would find it very practical to add more details for vacation accommodations. This would also apply in part to campsites and vacation apartments, but especially to hotels.
This would include details such as `sauna`, `swimming_pool`, `service:bicycle:rental`, `service:bicycle:parking`, `service:bicycle:washing`, `bar`. From my point of view, these are quite easy to identify on every visit.
Other, but somewhat more difficult tags would be `rooms` and `beds`.
And of course, `stars`. The wiki mentions that the classification in hotel stars is protected by copyright. But if it says “Hotel *** XYZ” on the building front, it should be legally recordable.

Now I'm asking myself whether it makes sense to create a quest for every single tag. These are usually yes/no tags. Or whether you can select all the relevant values in one go, similar to the recycling quest. In my opinion, the only advantage would be the reduction in the total number of quests available.

What do you think about adding such details with SCEE via quests at all? And what do you think about single quests vs. a multi-quest?

Thanks and Cheers!

**Comment by Helium314** [comment]
> What do you think about adding such details with SCEE via quests at all?

I think quests are the way to go, as long as the quests don't violate the no-spam guideline (on first glance most of them should be fine).

> And what do you think about single quests vs. a multi-quest?

While I would like to avoid large amounts of additional quests for SCEE, creating the form for such a multi-quest would be much more effort than simply adding a bunch of quests, and I don't think that's worth it. Maybe if it would be done in an abstract / re-usable way...

**Reply by mnalis** [reply]
I agree **single-topic quest are preferable**, even if there are quite a few of them. After all, people interested in using sauna might not be interested / inquire about bicycle washing service and other things in order to be able to solve a quest (IOW multi-topic quest would be much harder to solve, and unappealing to many mappers) ... 

Asking for **`stars` also seems fine to me**, there are no copyright/licensing issues if you're not copying from their database but instead creating your own database entering on-the-ground public data. (i.e. you may not copy a phonebook, but you can go from door to door and ask people if they'd share there name and phone number, and publish your own phonebook compile from such data).
